tests/sentry/api/endpoints/test_organization_dynamic_sampling_sdk_versions.py

```diff
@@ -175,6 +175,17 @@ def mocked_discover_query():
                 'count_if(transaction.source, notEquals, "")': 0,
                 "count()": 1,
             },
+            # project: spring
```

Comment:
Wasn't sure if we wanted to add these here. Also not sure about version number. Should it be `6.5.0` (without beta) instead?

Reply:
it does not matter, versions are not used for the comparison.